### PR TITLE
EZP-31186: Change widgets layout

### PR DESCRIPTION
**Pull request by GrabowskiM**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-31186
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Change widgets layout.
Also modified:
https://github.com/ezsystems/date-based-publisher/pull/149
https://github.com/ezsystems/ezplatform-workflow/pull/107
https://github.com/ezsystems/ezplatform-page-builder/pull/474

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/translations/locationview.en.xliff [hunk 1]
```diff
@@ -18,22 +18,22 @@
       </trans-unit>
       <trans-unit id="a4b9e9b4898dd285237fc60130970ce7ed988871" resname="content.create.choose_content_type">
         <source>Create content</source>
-        <target state="new">Create content</target>
+        <target state="new">Create your content</target>
```
Comment (DominikaK) [inline src/bundle/Resources/translations/locationview.en.xliff]:
```suggestion
        <target state="new">Create content</target>
```

File: src/bundle/Resources/views/themes/admin/content/widget/content_create.html.twig [hunk 2]
```diff
@@ -1,18 +1,18 @@
 {% trans_default_domain 'locationview' %}
 
-<div class="ez-extra-actions ez-extra-actions--create ez-extra-actions--hidden bg-secondary" data-actions="create">
-    <div class="ez-extra-actions__header">{{ 'content.create.choose_content_type'|trans|desc('Create content') }}</div>
+<div class="ez-extra-actions ez-extra-actions--create ez-extra-actions--hidden" data-actions="create">
+    <div class="ez-extra-actions__header">{{ 'content.create.choose_content_type'|trans|desc('Create your content') }}</div>
```
Comment (DominikaK) [inline src/bundle/Resources/views/themes/admin/content/widget/content_create.html.twig]:
```suggestion
    <div class="ez-extra-actions__header">{{ 'content.create.choose_content_type'|trans|desc('Create content') }}</div>
```

**Comment by GrabowskiM** [comment]
Selection_017
Selection_018
Selection_019

File: src/bundle/Resources/public/scss/_extra-actions.scss [hunk 3]
```diff
@@ -175,32 +163,80 @@
...
+            .ez-extra-actions {
+                &--hidden {
+                    transform: translate(calc(-100% - #{calculateRem(15px)}), 0) scaleX(0);
+                }
+            }
         }
     }
 }
+
+.ez-btn {
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_extra-actions.scss]: Wrong place, should be in `buttons.scss`

File: src/bundle/Resources/public/scss/_extra-actions.scss [hunk 4]
```diff
@@ -175,32 +163,80 @@
...
+    &--active-button {
+        background-color: $ez-white;
+        color: $ez-color-secondary;
+        border-color: $ez-white;
+
+        .ez-icon {
+            fill: $ez-color-secondary;
+        }
+    }
+}
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_extra-actions.scss]: missing empty line at the end of file

File: src/bundle/Resources/public/scss/_extra-actions.scss [hunk 5]
```diff
@@ -42,33 +43,22 @@
...
-        line-height: calculateRem(45px);
-        color: $ez-white;
-        padding: 0 calculateRem(8px);
-        font-weight: 700;
+        line-height: calculateRem(50px);
+        font-size: calculateRem(17px);
+        color: $ez-color-secondary;
+        margin-bottom: calculateRem(15px);
+        font-weight: 600;
+        border-bottom: 1px solid $ez-color-secondary;
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_extra-actions.scss]: `calculateRem`

File: src/bundle/Resources/public/scss/_extra-actions.scss [hunk 6]
```diff
@@ -162,8 +162,30 @@
...
-    &--dropdown {
+        &--hidden {
+            opacity: 0;
+            transform: translate(calc(-100%), 0) scaleX(0);
+            transition: all 0.2s ease-out;
+        }
+    }
+}
+
+.ez-dropdown {
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_extra-actions.scss]: admin-ui doesn't know about page-builder. This cannot be style here. Should be in page-builder.

**Pull request merged**